.eslintrc.js

```diff
@@ -95,6 +95,7 @@ module.exports = {
         '@typescript-eslint/no-unnecessary-condition': 'off',
         '@typescript-eslint/consistent-type-imports': ['error', { prefer: 'type-imports' }],
         '@typescript-eslint/unbound-method': 'off',
+        '@typescript-eslint/sort-type-union-intersection-members': 'off',
```

Comment:
this auto-sorting of union keys was annoying a lot.

For eg: If I type following thing `const HeadingVariants = 'small' | 'medium' | 'large' | 'subheading'` it will sort the union keys in alphabetical order but I wanted it to be the way I declare because it's easier to read that way and most of the cases devs do this so turning off this rule

Reply:
Yeah mostly find this rule useless.